mappings/net/minecraft/item/HoeItem.mapping

```diff
@@ -9,12 +13,19 @@ CLASS net/minecraft/class_1794 net/minecraft/item/HoeItem
 		ARG 1 p
 	METHOD method_36984 (Lnet/minecraft/class_2680;Lnet/minecraft/class_1838;)V
 		ARG 1 context
-	METHOD method_36985 getTillingConsumer (Lnet/minecraft/class_2680;Lnet/minecraft/class_1935;)Ljava/util/function/Consumer;
-		ARG 0 state
-		ARG 1 dropItem
+	METHOD method_36985 getTillingAction (Lnet/minecraft/class_2680;Lnet/minecraft/class_1935;)Ljava/util/function/Consumer;
```

Comment:
I suggest `createTillAndDropAction` and `createTillAction` for the two consumer factories; on reading, I thought they were getting the tilling action from the map (lol)